mobile_verifier/src/reward_shares.rs

```diff
@@ -236,7 +247,32 @@ impl MapperShares {
                         reward: Some(ProtoReward::SubscriberReward(subscriber_reward)),
                     },
                 )
+            });
+
+        let verified_rewards = self
+            .verified_mapping_event_shares
+            .into_iter()
+            .map(move |verified_share| proto::SubscriberReward {
+                subscriber_id: verified_share.subscriber_id,
+                discovery_location_amount: (Decimal::from(verified_share.total_reward_points)
```

Comment:
We should have a separate entry for verification mapping rather than including all in discover_location_amount.  Ie we should have a second entry in subscriber reward such as verification_mapping_amount for which the value is the subscribers reward for verification mapping for the epoch

Reply:
I think you may have misunderstood my suggestion above.  Rather than producing two SubscriberRewards ( one for disco mapping and then one for verification mapping ) the two should be combined into a single SubscriberReward but with distinct fields within the proto.  The subscriber reward proto itself would be updated to something like:

message subscriber_reward {
  // id of the subscriber to which the reward will be credited
  bytes subscriber_id = 1;
  /// Amount in bones credited to the subscriber for location sharing
  uint64 discovery_location_amount = 2;
  /// Amount in bones credited to the subscriber for verification mapping 
  uint64 verification_mapping_amount = 3;
}

Reply:
I think this https://github.com/helium/oracles/pull/839/commits/63dad9aa2d02cdf74220663e53e38f95482611fa does it?